content/docs/learning-guide/tutorials/environments/create-terrain-from-images.md

```diff
@@ -0,0 +1,314 @@
+---
+linkTitle: Create Terrain from Images
+title: Create Terrain from Images
+description: Learn to create terrain in Open 3D Engine (O3DE) using two methods - by directly creating the entities and by using Landscape Canvas to create the entities.
+weight: 300
+toc: true
+---
+
+Terrain in Open 3D Engine (O3DE) is generated from *gradients*. Gradients, in the context of this tutorial, represent changes in elevation and weight masks for materials. Gradients can be generated procedurally or processed from an image. In this tutorial, you'll create terrain elevation and apply detail materials using image gradients, and add support for PhysX simulations with a heightfield collider and physics materials.
+
+This tutorial requires that you have the **Terrain**, **Landscape Canvas**, and **PhysX** Gems enabled in your project.
+
+## Required image asset
+
+To complete this tutorial, you need a *heightmap* that's similar to the following example:
```

Comment:
I think you should just instruct people to save this example heightmap to their project Assets folder so they can use it in this tutorial. If they want to use a different one, they'll do so anyway.

Reply:
This image is to demonstrate what a heightmap looks like. It shouldn't be used for the tutorial which is why the tutorial includes information on the technical details of heighmaps and processing them. A heightmap asset (even a small one) can be several megabytes in size and I don't have a way to distribute those assets.

Reply:
> It shouldn't be used for the tutorial [...]

Just FYI, I did a "save image as" on the heightmap image in the preview page and it worked fine for the first half of the tutorial. This is also what I was hoping you might be able to do for some of the other assets in the second half, if they can't be found somewhere in the o3de installation.